src/components/illustration/illustration.tsx

```diff
@@ -56,7 +60,10 @@ export const Illustration = (props: IllustrationProps) => {
   const IllustrationSvg = IllustrationSet[props.show];
 
   return (
-    <IllustrationLayout size={props.size ?? IllustrationSize.NORMAL}>
+    <IllustrationLayout
+      className={props.className}
+      size={props.size ?? IllustrationSize.NORMAL}
+    >
```

Comment:
I need to directly wrap this component for the Accordion implementation. I think it's never wrong to be a good citizen of the ecosystem and by supporting a famous framework like styled components (but also others using the same technique).

I could have prevented this by wrapping the Illustration component in an additional styled div. Much easier now. 
Find `const PanelIllustration = styled(Illustration)` in the code to understand.